sql/moz-fx-data-shared-prod/fenix_derived/meta_attribution_country_counts_v1/metadata.yaml

```diff
@@ -0,0 +1,28 @@
+friendly_name: Meta Attribution Country Counts
+description: |-
+  Aggregated daily country counts of Meta attribution data.
+
+  This table should be accessed through the user-facing view
+  `fenix.meta_attribution_country_counts`.
```

Comment:
You'll also need to create this view. Usually it's autogenerated when you use `bqetl query create`, but you can use `bqetl view create fenix.meta_attribution_country_counts`

Reply:
Fixed, added